.gitignore

```diff
@@ -1,4 +1,6 @@
 Cargo.lock
 target/
 .cargo/
-.hc
\ No newline at end of file
+.hc
+workdir/dna/demo-dna.dna
+workdir/happ/demo-happ.happ
```

Comment:
if you gitgnore these, it would also make sense to remove them from the git repo, and also the README would have to be updated a bit, since it (silly) assumes those files are coming in pre-built pretty much

Reply:
Yep, I did do that!